templates/terraform/sweeper_file.go.erb

```diff
@@ -0,0 +1,115 @@
+<%= lines(autogen_notice :go) -%>
+
+package google
+
+import (
+  "fmt"
+  "testing"
+  "strings"
+  "github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+  "github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+<%
+sweeper_name = product_ns + object.name
+wrap_path = object&.nested_query&.keys&.first || object.collection_url_key
+-%>
+
+func init() {
+	resource.AddTestSweepers("<%= sweeper_name -%>", &resource.Sweeper{
+		Name: "<%= sweeper_name -%>",
+		F:    testSweep<%= sweeper_name -%>,
+	})
+}
+
+func testSweep<%= sweeper_name -%>(region string) error {
+	resource_name := "<%= sweeper_name -%>"
+	log.Printf("[SWEEP_LOG] Sweeping %s",resource_name)
+
+	config, err := sharedConfigForRegion(region)
+	if err != nil {
+		log.Fatalf("[SWEEP_LOG] error getting shared config for region: %s", err)
+	}
+
+	err = config.LoadAndValidate(context.Background())
+	if err != nil {
+		log.Fatalf("[SWEEP_LOG] error loading: %s", err)
+	}
+
+	list_template := strings.Split("<%= object.__product.base_url + object.base_url -%>","?")[0]
+
+	d := &ResourceDataMock{
+		FieldsInSchema: map[string]interface{}{
+			"project":config.Project,
+			"region":region,
+			"location":region,
+			"zone":"-",
```

Comment:
Does sending `-` as the zone work for zonal resources? I saw it does for GKE, but I'm doubtful it will for GCE resources like Disk.

If it does, would a similar value work for regions?

Reply:
At least, the API explorer rejected me trying `-`.

Reply:
At least from the resources I've seen so far, if there is not an aggregatedList call option available, the "-" zone will return all. [GCE has aggregatedList available](https://cloud.google.com/compute/docs/reference/rest/v1/disks/aggregatedList), which I plan to tackle in another PR as the response is structured differently